numpy/ma/tests/test_extras.py

```diff
@@ -1589,5 +1589,69 @@ def test_shape_scalar(self):
         assert_equal(b.mask.shape, b.data.shape)
 
 
+class TestStack(object):
+
+    def test_stack_1d(self):
+        a = masked_array([0, 1, 2], mask=[0, 1, 0])
+        b = masked_array([9, 8, 7], mask=[1, 0, 0])
```

Comment:
A test where `mask=False` would be good somewhere - perhaps not here.

Reply:
I just added that test; also works fine.